database/index.js

```diff
@@ -14,19 +14,32 @@ connection.connect((err) => {
   console.log('Connection to database established');
 });
 
-const insertRecords = (data) => {
-  connection.query('INSERT INTO roomlist (roomname, price, numberOfBedrooms, rating, numberOfReviews, urlToImage, roomType, instantBook) VALUES ?', [data]);
+const insertRoomlistRecords = (data) => {
+  connection.query('INSERT INTO roomlist (roomname, price, numberOfBedrooms, rating, numberOfReviews, roomType, instantBook) VALUES ?', [data]);
 };
 
-const getAllRecords = (sendCallback) => {
+const insertImagesRecords = (data) => {
+  connection.query('INSERT INTO images (urlToImage, roomId) VALUES ?', [data]);
```

Comment:
can return this line to make it a promise and you can attach a callback/thenable later if you want